docs/content/guides/technical-specification/mobile-support.md

```diff
@@ -0,0 +1,113 @@
+| [Column moving](@/guides/columns/column-moving.md)                         |     &cross;      |   &cross;    |   &cross;    |
+| [Column freezing](@/guides/columns/column-moving.md)                       |     &cross;      |   &cross;    |   &cross;    |
+| [Manual column resizing](@/guides/columns/column-width.md)                 |     &cross;      |   &cross;    |   &cross;    |
+| [Column summary](@/guides/columns/column-summary.md)                       |     &check;      |   &check;    |   &check;    |
+| [Column virtualization](@/guides/columns/column-virtualization.md)         |     &check;      |   &check;    |   &check;    |
+| [Column menu](@/guides/columns/column-menu.md)                             |     &check;      |   &check;    |   &check;    |
+| [Column filter](@/guides/columns/column-filter.md)                         |     &check;      |   &check;    |   &check;    |
+| [Row headers](@/guides/rows/row-header.md)                                 |     &check;      |   &check;    |   &check;    |
+| [Row parent-child](@/guides/rows/row-parent-child.md)                      |     &check;      |   &check;    |   &check;    |
+| [Row hiding](@/guides/rows/row-hiding.md)                                  |     &check;      |   &check;    |   &check;    |
+| [Row moving](@/guides/rows/row-moving.md)                                  |     &cross;      |   &cross;    |   &cross;    |
+| [Row freezing](@/guides/rows/row-freezing.md)                              |     &cross;      |   &cross;    |   &cross;    |
+| [Manual row resizing](@/guides/rows/row-height.md)                         |     &cross;      |   &cross;    |   &cross;    |
+| [Row virtualization](@/guides/rows/row-virtualization.md)                  |     &check;      |   &check;    |   &check;    |
+| [Rows sorting](@/guides/rows/row-sorting.md)                               |     &check;      |   &check;    |   &check;    |
+| [Row trimming](@/guides/rows/row-trimming.md)                              |     &check;      |   &check;    |   &check;    |
+| [Row pre-populating](@/guides/rows/row-prepopulating.md)                   |     &cross;      |   &cross;    |   &cross;    |
+| [Clipboard](@/guides/cell-features/clipboard.md)                           |     &check;      |   &check;    |   &check;    |
+| [Merge cells](@/guides/cell-features/merge-cells.md)                       |     &check;      |   &check;    |   &check;    |
+| [Conditional formatting](@/guides/cell-features/conditional-formatting.md) |     &check;      |   &check;    |   &check;    |
+| [Text alignment](@/guides/cell-features/text-alignment.md)                 |     &check;      |   &check;    |   &check;    |
+| [Disabled cells](@/guides/cell-features/disabled-cells.md)                 |     &check;      |   &check;    |   &check;    |
+| [Comments](@/guides/cell-features/comments.md)                             |     &cross;      |   &cross;    |   &cross;    |
```

Comment:
This can be marked as "checked" . I test it on Android using the context menu and during testing documentation on iOS.